backend/src/app.module.ts

```diff
@@ -1,10 +1,31 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { PostsController } from './posts/posts.controller';
+import { CommentsController } from './comments/comments.controller';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { join } from 'path';
+import { Connection } from 'typeorm';
+import { UsersModule } from './users/users.module';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'mysql',
+      host: 'db',
+      port: 3306,
+      username: 'root',
+      password: 'p@ssw0rd',
```

Comment:
this one too, please use .env if possible

Reply:
In nestjs, 
1. you can create a folder named config and put .env in the folder.  
2. add `require('dotenv').config({path: __dirname + '/config/.env'})` in the code.
3. call the value like **process.env.DB_PASSWORD** 

PS1. please ignore **.env** file
PS2. you can see this as ref. https://github.com/2110423-2019-2/sec1_TerkyCompany-BackEnd/blob/master/src/main.ts

Reply:
Fixed